integration-cli/test_vars_unix.go

```diff
@@ -0,0 +1,8 @@
+// +build !windows
```

Comment:
I see that in standard library tags:
```
// +build darwin dragonfly freebsd linux nacl netbsd openbsd solaris
```
treated as `unix`, but maybe `!windows` is the same :) apart from android tag.